Wallet Download and Installation.md

```diff
@@ -0,0 +1,64 @@
+# Downloading and Installing Wallets to Your Devices
+
+Visit the **[Downloads](https://www.skycoin.net/downloads/)** page from the Skycoin Website to download the Skycoin Desktop Wallet and Skycoin Mobile Wallet for the device you are using:
+
+## [Desktop Wallet](Desktop_Wallet)
+
+* ### [Windows](Windows)
+* ### [MAC OS](MAC_OS)
```

Comment:
I think Mac OS or MacOS is the right way to spell it. Please refer to Apples official sources and update the spelling.